NetSparkle/AppCast.cs

```diff
@@ -170,155 +190,175 @@ private bool ReadStream(Stream inputstream, string signature)
+        //                        {
+        //                            currentItem.OperatingSystemString = operatingSystem;
+        //                        }
+
+        //                        string mimeType = reader.GetAttribute(typeAttribute);
+        //                        if (mimeType != null && mimeType != "")
+        //                        {
+        //                            currentItem.MIMEType = mimeType;
+        //                        }
+        //                    }
+        //                    break;
+        //                case pubDateNode:
+        //                    if (currentItem != null)
+        //                    {
+        //                        // "ddd, dd MMM yyyy HH:mm:ss zzz" => Standard date format
+        //                        //      e.g. "Sat, 26 Oct 2019 22:05:11 -05:00"
+        //                        // "ddd, dd MMM yyyy HH:mm:ss Z" => Check for MS AppCenter Sparkle date format which ends with GMT
+        //                        //      e.g. "Sat, 26 Oct 2019 22:05:11 GMT"
+        //                        // "ddd, dd MMM yyyy HH:mm:ss" => Standard date format with no timezone (fallback)
+        //                        //      e.g. "Sat, 26 Oct 2019 22:05:11"
+        //                        string[] formats = { "ddd, dd MMM yyyy HH:mm:ss zzz", "ddd, dd MMM yyyy HH:mm:ss Z", "ddd, dd MMM yyyy HH:mm:ss" };
+        //                        string dt = reader.ReadString().Trim();
+        //                        if (DateTime.TryParseExact(dt, formats, System.Globalization.CultureInfo.InvariantCulture, DateTimeStyles.None, out DateTime dateValue))
+        //                        {
+        //                            _logWriter.PrintMessage("Converted '{0}' to {1}.", dt, dateValue);
+        //                            currentItem.PublicationDate = dateValue;
+        //                        }
+        //                        else
+        //                        {
+        //                            _logWriter.PrintMessage("Cannot parse item datetime {0}", dt);
+        //                        }
+        //                    }
+        //                    break;
+        //            }
+        //        }
+        //        else if (reader.NodeType == XmlNodeType.EndElement)
+        //        {
+        //            switch (reader.Name)
+        //            {
+        //                case itemNode:
+        //                    Items.Add(currentItem);
+        //                    break;
+        //            }
+        //        }
+        //    }
+
+        //    // sort versions in reverse order
+        //    Items.Sort((item1, item2) => -1 * item1.CompareTo(item2));
+        //}
+
         /// <summary>
         /// Returns sorted list of updates between current and latest. Installed is not included.
         /// </summary>
-        public List<AppCastItem> GetUpdates()
```

Comment:
Can we change this back to a `List`, please? `GenerateAppCastXml` takes a `List`. Was there another reason I'm missing for this to change back to a `[ ]`?

Reply:
Not sure what happened there, it should be List everywhere. Must have happened when I merged from upstream I think.